lib/views/admin_request/hidden_user_explanation/_personal_information.text.erb

```diff
@@ -1,3 +1,20 @@
 <%= _('We consider it is not a valid FOI request as it was correspondence ' \
       'about your personal circumstances, and we have therefore hidden it ' \
       'from other users.') %>
+
+<%= _('* If you would like to know what information a public authority holds ' \
+      'about you, you should make a “Subject Access Request” in private ' \
+      'Data Protection law. The Information Commissioner’s website provides '\
+      'advice on accessing your own personal information: ' \
+      'https://ico.org.uk/your-data-matters/your-right-of-access') %>
+
+<%= _('* It includes extraneous material such as allegations, personal ' \
+      'details, personal opinions. FOI requests should take the form of ' \
+      'a simple and concise request for information -- see our help pages ' \
+      'for more information on how to format your request: ' \
+      'https://www.whatdotheyknow.com/help/requesting Personal or biased ' \
```

Comment:
This should use a URL helper.

I think it also needs a line break after the URL, or a full stop at least.

Reply:
A line break. The problem with a full stop is going to be people mistakenly including it as part of the address.

Reply:
No, wait - a full stop. It looks kinda broken with a line break.